src/wp-includes/rest-api/endpoints/class-wp-rest-users-controller.php

```diff
@@ -1131,7 +1147,7 @@ public function check_user_password( $value, $request, $param ) {
 			return new WP_Error( 'rest_user_invalid_password', __( 'Passwords cannot be empty.' ), array( 'status' => 400 ) );
 		}
 
-		if ( false !== strpos( $password, "\\" ) ) {
+		if ( false !== strpos( $password, '\\' ) ) {
 			return new WP_Error( 'rest_user_invalid_password', __( 'Passwords cannot contain the "\\" character.' ), array( 'status' => 400 ) );
```

Comment:
This unchanged line has `"\\"` inside of a single quoted string. This seems wrong.

Reply:
Seems it is a literal comparison, we match upstream
https://github.com/WordPress/wordpress-develop/blob/trunk/src/wp-includes/rest-api/endpoints/class-wp-rest-users-controller.php#L1311

Reply:
You say "comparison", but that's the changed line. which I already mentioned in the previous comment. 
This comment is about the unchanged line (return statement), which has a single quoted string containing a double quoted string of `"\\"`. It just seems weird that it would be coded intentionally that way when I know there was an auto formatter run on the code.

Reply:
I was misreading it. My mistake.